nvim-plugin/lua/ethersync/cursor.lua

```diff
@@ -248,12 +248,17 @@ function M.track_cursor(bufnr, callback)
     vim.api.nvim_exec_autocmds("CursorMoved", {})
 end
 
-function M.unfollow_cursor()
+local on_key_nsid = nil
+local function unfollow_cursor()
     if following_user_id ~= nil then
         print("Unfollowed cursor")
         following_user_id = nil
+        vim.on_key(nil, on_key_nsid)
     end
 end
+function M.unfollow_cursor()
```

Comment:
You're already anticipating removing the public function here, right? I like it!

Reply:
(And, after we've tried it out a bit in practice, I would probably favor doing so!)